tests/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+include(FetchContent)
+
+FetchContent_Declare(doctest
+  GIT_REPOSITORY https://github.com/onqtam/doctest.git
+  GIT_TAG master)
+FetchContent_MakeAvailable(doctest)
 
```

Comment:
Please make sure that doctest is not pulled when the tests are not enabled. That is, by default, it should not get pulled.

Reply:
That comes free from the main CMakeLists.txt:
```cmake
if(FASTFLOAT_TEST)
  enable_testing()
  add_subdirectory(tests)
endif(FASTFLOAT_TEST)
```

Reply:
Then we are good. :-)